spec/tests/ColumnLineageDatasetFacet/1.json

```diff
@@ -1,30 +1,47 @@
 {
   "columnLineage": {
     "fields": {
-      "order_day_of_week": {
+      "NAME": {
         "inputFields": [
           {
-            "field": "order_placed_on",
-            "name": "public.top_delivery_times",
-            "namespace": "postgres://postgres:5432"
-          }
-        ],
-        "transformationDescription": "",
-        "transformationType": ""
-      },
-      "order_placed_on": {
-        "inputFields": [
+            "namespace": "SnowflakeOpenLineage",
+            "name": "CUSTOMERS",
+            "field": "NAME",
+            "transformations": [
+              {
+                "type": "DIRECT",
+                "masking": false
+              }
+            ]
+          },
+          {   
+            "namespace": "SnowflakeOpenLineage",
+            "name": "CUSTOMERS",
+            "field": "ID",
+            "transformations": [
+              { 
+                "type": "INDIRECT",
+                "subtype": "FILTER",
```

Comment:
in case of joins the subtype should be `JOIN`

Reply:
Fixed! (not sure if I should resolve this convo or if you should after re-review?)

I also added the other 3 columns so the example is complete.